### PR TITLE
TRIVIAL: Update changelog for 7.4.0

### PR DESCRIPTION
**Pull request by strouhaljan**
---

[Check PR owner responsibilities](https://confluence.intgdc.com/display/Development/Code-reviews#Code-reviews-Ownerresponsibilities)

Supported PR commands:

Command | Description
--- | ---
`extended test - examples` | Live examples tests
`extended test - storybook` | Storybook screenshot tests

See more [options](https://confluence.intgdc.com/display/kbhr/How+to+work+with+git+and+Github#HowtoworkwithgitandGithub-Extendedchecks).

# PR Checklist

- [ ] Verify code changes ([Checklist](https://confluence.intgdc.com/display/Development/Code-reviews+checklist), [Best practices](https://confluence.intgdc.com/display/Development/Code-reviews+best+practices))
- [ ] [Verify pull-request formalities](https://confluence.intgdc.com/display/Development/Code-reviews)
- [ ] Change was tested by using [gdc-dev-release](https://confluence.intgdc.com/display/~tomas.vojtasek/Private+NPM) in [gdc-analytical-designer](https://github.com/gooddata/gdc-analytical-designer) and [gdc-dashboards](https://github.com/gooddata/gdc-dashboards) (if applicable)
- [ ] Migration guide (for major changes) is mentioned in [CHANGELOG.md](../blob/master/CHANGELOG.md).
- [ ] Successful `extended test - examples`
- [ ] Successful `extended test - storybook`
- [ ] Checked yarn.lock consistency (no dep. duplicities especially Goodstrap)

# Related PRs

- gdc-analytical-designer:
- gdc-dashboards:

# Related Jira tasks

### CODE REVIEW

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/6615/ : SUCCESS in 10m 02s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-oss-check/2447/ : SUCCESS in 3s

File: CHANGELOG.md [hunk 1]
```diff
@@ -21,6 +21,7 @@ The REST API versions in the table are just for your information as the values a
 
 ### Added
 
+- new visualization type - Bullet chart
```
Comment (evgeniabogranskaya) [inline CHANGELOG.md]: Bullet chart component ([doc](https://sdk.gooddata.com/gooddata-ui/docs/bullet_chart_component.html))

File: CHANGELOG.md [hunk 2]
```diff
@@ -21,6 +21,7 @@ The REST API versions in the table are just for your information as the values a
...
 - Column auto-sizing in pivot tables can now be configured (applied to visible columns during initial table load).
```
Comment (evgeniabogranskaya) [inline CHANGELOG.md]: Column auto-sizing in pivot tables (applied to visible columns during the initial table load)

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/6618/ : SUCCESS in 12m 59s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-oss-check/2449/ : SUCCESS in 3s

**Comment by xMort** [comment]
LGTM

**Comment by gdgate** [comment]
Build failed (gate pipeline).

- gooddata-react-components-storybook-zuul-docker : SKIPPED
- gooddata-react-components-unit-tests-zuul-docker-el7 : SKIPPED
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-merge-compliance-check/1598/ : FAILURE in 13s
- gooddata-react-components-examples-zuul-docker : SKIPPED

**Comment by leosflodr** [comment]
approved by DC

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-storybook-zuul-docker/3446/ : SUCCESS in 17m 36s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/6619/ : SUCCESS in 9m 15s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-merge-compliance-check/1599/ : SUCCESS in 22s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-examples-zuul-docker/2300/ : SUCCESS in 18m 05s

**Pull request merged**